src/common/datetime.ts

```diff
@@ -52,3 +53,22 @@ export const parseDateTime = (dateString: string | undefined): string =>
         .tz(dateString, "Asia/Singapore")
         .format("M/D/YY h:mm:ss A")} GMT+08:00`
     : "";
+
+/**
+ * Returns a date string ignoring fields with only zeroes
+ * @param "2020-00-00 or 2020-02-00"
+ * @returns "2020 or 2020-20"
+ */
+export const parseDateWithoutZeroes = (dateString: string): string => {
+  // To parse Vault dob which is always in YYYY-MM-DD format
+  const dayIsZero = dateString.slice(8, 10) === ZEROES;
```

Comment:
should this use slice or better use split by hyphen, i think split cleaner

Reply:
i thinking incase if the provider still use stupid date formats like 1980-FEB-01 
then it will cause error here

Reply:
okay changed